aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/FeatureAssert.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws;
+
+import com.amplifyframework.predictions.models.Feature;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Utility class to assert that two features have matching values
+ * without regarding their IDs or confidence scores.
+ */
+final class FeatureAssert {
+    private FeatureAssert() {}
+
+    /**
+     * Assert that a given value equals the feature's assigned value.
+     * @param expectedValue the value to expect in a feature
+     * @param actual the actual feature
+     * @param <T> the type of expected value
+     */
+    static <T> void assertMatches(T expectedValue, Feature<T> actual) {
```

Comment:
Maybe this could be named _`assertFeatureValue(T expectedValue, Feature<T> actualFeature)`_?

Reply:
haha, I feel like there should be a verb to make grammatical sense (i.e. "matches"). I was actually considering `assertValueMatches` or `assertFeatureValueMatches`, but I thought that was a bit too long. Besides, this method is static, so calling `FeatureAssert` first would imply that we are asserting to see if features match.